src/sidebar/components/HelpPanel.js

```diff
@@ -25,17 +25,17 @@ import VersionInfo from './version-info';
  */
 function HelpPanelTab({ linkText, url }) {
   return (
-    <div className="help-panel-tabs__tab">
+    <div className="HelpPanel-tabs__tab">
```

Comment:
This one looks off.

Should be this `HelpPanelTab__tab` or perhaps just `HelpPanelTab`